--- setup.py	Tue Sep 30 02:42:28 2014
+++ setup - Copy.py	Mon May 18 18:26:17 2015
@@ -92,10 +92,10 @@
                '%(linux2)s/include/linux' %(JDK)],
     'sunos5': ['%(sunos5)s/include' %(JDK),
                '%(sunos5)s/include/solaris' %(JDK)],
-    'win32': ['%(win32)s/include' %(JDK),
-              '%(win32)s/include/win32' %(JDK)],
-    'mingw32': ['%(mingw32)s/include' %(JDK),
-                '%(mingw32)s/include/win32' %(JDK)],
+    'win32': ['%(win32)s\\include' %(JDK),
+              '%(win32)s\\include\\win32' %(JDK)],
+    'mingw32': ['%(mingw32)s\\include' %(JDK),
+                '%(mingw32)s\\include\\win32' %(JDK)],
     'freebsd7': ['%(freebsd7)s/include' %(JDK),
                  '%(freebsd7)s/include/freebsd' %(JDK)],
 }
@@ -142,7 +142,7 @@
     'sunos5': ['-L%(sunos5)s/jre/lib/i386' %(JDK), '-ljava',
                '-L%(sunos5)s/jre/lib/i386/client' %(JDK), '-ljvm',
                '-R%(sunos5)s/jre/lib/i386:%(sunos5)s/jre/lib/i386/client' %(JDK)],
-    'win32': ['/LIBPATH:%(win32)s/lib' %(JDK), 'Ws2_32.lib', 'jvm.lib'],
+    'win32': ['/LIBPATH:%(win32)s\\lib' %(JDK), 'Ws2_32.lib', 'jvm.lib'],
     'mingw32': ['-L%(mingw32)s/lib' %(JDK), '-ljvm'],
     'freebsd7': ['-L%(freebsd7)s/jre/lib/i386' %(JDK), '-ljava', '-lverify',
                  '-L%(freebsd7)s/jre/lib/i386/client' %(JDK), '-ljvm',
@@ -169,8 +169,8 @@
     'ipod': ['jikes', '-cp', '/usr/share/classpath/glibj.zip'],
     'linux2': ['javac'],
     'sunos5': ['javac'],
-    'win32': ['%(win32)s/bin/javac.exe' %(JDK)],
-    'mingw32': ['%(mingw32)s/bin/javac.exe' %(JDK)],
+    'win32': ['%(win32)s\\bin\\javac.exe' %(JDK)],
+    'mingw32': ['%(mingw32)s\\bin\\javac.exe' %(JDK)],
     'freebsd7': ['javac'],
 }
 
@@ -179,8 +179,8 @@
     'ipod': [],
     'linux2': ['javadoc'],
     'sunos5': ['javadoc'],
-    'win32': ['%(win32)s/bin/javadoc.exe' %(JDK)],
-    'mingw32': ['%(mingw32)s/bin/javadoc.exe' %(JDK)],
+    'win32': ['%(win32)s\\bin\\javadoc.exe' %(JDK)],
+    'mingw32': ['%(mingw32)s\\bin\\javadoc.exe' %(JDK)],
     'freebsd7': ['javadoc'],
 }
 
